packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -463,7 +463,10 @@ export const TooltipPlugin2 = ({
     config.addHook('setData', (u) => {
       yZoomed = false;
       yDrag = false;
-      dismiss();
+
+      if (_isPinned) {
```

Comment:
we probably should do something better for the anchored case, too. but that's more difficult, since the current design is to ensure that the tooltip represents the hovered data under the cursor.